pkg/boot/linux/load_linux_other.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build linux && !amd64 && !arm64
+// +build linux,!amd64,!arm64
```

Comment:
so can you call this maybe
load_other_linux.go and then have a build constraint for !amd64,!arm64?

you're not following the naming convention when you put linux in the middle of the name.

Reply:
Done. 

You are right...in such case, go build resort to match for either a knownOS, or a knownArch against last segment. https://cs.opensource.google/go/go/+/refs/tags/go1.18.3:src/go/build/build.go;l=1986-1988;drc=2e773a3894fba7af744090d7d42968f4993018e2.